labelCloud/view/gui.py

```diff
@@ -238,6 +240,13 @@ def __init__(self, control: "Controller") -> None:
         self.connect_events()
         self.set_checkbox_states()  # tick in menu
 
+        # Run startup dialog
+        startup_dialog = StartupDialog()
+        if startup_dialog.exec():
+            startup_dialog.save_class_labels()
+        else:
+            sys.exit()
+
```

Comment:
```suggestion
self.controller.startup(self)
```
The controller needs to start up after the startup dialog is finished to have the updated class labels.

Reply:
and remove L237